src/embind/embind.js

```diff
@@ -205,14 +205,11 @@ var LibraryEmbind = {
       return body.apply(this, arguments);
     };
 #else
-    /*jshint evil:true*/
-    return new Function(
-        "body",
-        "return function " + name + "() {\n" +
-        "    \"use strict\";" +
-        "    return body.apply(this, arguments);\n" +
-        "};\n"
-    )(body);
+    return {
+      [name](...args) {
+        return body(...args);
```

Comment:
Do you know why this #else block exists?  Why don't we just always use the simpler `DYNAMIC_EXECUTION == 0` form above?

Reply:
[I have encountered it for v8 on Cloudflare workers](https://github.com/NickCarducci/mastercard-backbank/tree/main/src) when it was trying to tell me the modularized Module has no default export.

Reply:
But do you know why we can't just completely remove the `if/else` here?  (and just use what is in the of block now in all cases?)

Reply:
Noope!

Reply:
Can you try uploading a PR to remove the if/else then?

Reply:
I could, but it would be a favor not something I need. I believe a better PR would be to map an array of arguments instead of[ using] the spread operator. I can do that and achieve absolutely unlimited arguments within the week! Type Y to confirm.

Reply:
I'm not quite sure what you mean by "map an array of arguments" or "absolutely unlimited arguments".  Perhaps the best thing to do would be upload a proposed change.

In the mean time I might look into removing the if/else as a intermediate step.

Reply:
I am also only aware of the createNamedFunction being used for error messages from the output Module.

Reply:
Actually it looks like closure does a good job of transpiling these https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#computed_property_names to ES5 so I think your solution is good.

Any it works for DYNAMIC_EXECUTION so if you want to update this PR to replace the whole if/else that would work.. or I'm happy to do that too.

Reply:
wow
a week flys by when you are having fun and almost embarrassed
does this other pull request work? should I truly not be editing this one? I haven't done these before
[`{[name]:function(arguments){return body.call(null,arguments)}}[name]`](https://github.com/emscripten-core/emscripten/compare/main...NickCarducci:patch-1)

Reply:
You can edit existing PRs, or open new ones.  Its up to you.